LSDR/Assets/Scripts/Audio/MusicSystem.cs

```diff
@@ -15,6 +15,7 @@ public class MusicSystem : ScriptableObject
         public string CurrentSong { get; private set; }
         public string CurrentArtist { get; private set; }
         
+        //DEPRECATED: Torii implementation
```

Comment:
This code isn't deprecated, it's in use. All instances of things in the 'Torii' namespace aren't deprecated - I'd built up a lot of useful utility functions for Torii so I grouped them together. The Torii project has evolved from a level editor to a kind of framework for Unity now, so it's still heavily in use.

Reply:
Sorry about thinking torii was deprecated, i didn't know you were using it as a framework now.